2021q3/mgb.adoc

```diff
@@ -0,0 +1,25 @@
+== Microchip LAN743x mgb(4) Device Driver
+
+Links: +
+link:https://cgit.freebsd.org/src/commit/?id=8890ab7758b8a03a68f3fe596f6a5446921631a5[Commit adding mgb(4) driver] URL:  https://cgit.freebsd.org/src/commit/?id=8890ab7758b8a03a68f3fe596f6a5446921631a5 +
+link:https://cgit.freebsd.org/src/commit/?id=543df609072fe49079c36d6bee510e1645edde3a[Commit connecting mgb(4) module to the build] URL: https://cgit.freebsd.org/src/commit/?id=543df609072fe49079c36d6bee510e1645edde3a
+
+Contact: Ed Maste <emaste@freebsd.org>
+
+Microchip's LAN7430 and LAN7431 are PCIe Gigabit Ethernet interface ICs, with
+an integrated PHY (LAN7430) or RGMII interface (LAN7431).
+
+In 2019 Gerald ND Aryeetey, a FreeBSD Foundation intern, developed the mgb(4)
+driver for these devices.  It was added to the tree but not yet connected to
+the build.  Since it was committed a number of sweeping iflib changes were made,
+which included updates to mgb(4).
+
+I have addressed some outstanding issues in the driver, and have now added the
+module to the build.  It is available in -CURRENT snapshot images.  The driver
+is functional, although some additional work is still needed.  In particular,
+configuration of the Receive Filtering Engine, and VLAN and RX/TX checksum
+offload are required.  Caveats and notes are described in the man page.
```

Comment:
"manual page" (or better, "mgb(4)" with the appropriate AsciiDoc magic to linkify it). Otherwise, LGTM.